libs/SalesforceSDK/AndroidManifest.xml

```diff
@@ -47,7 +47,7 @@
             android:theme="@style/SalesforceSDK"
             android:launchMode="singleInstance"
             android:screenOrientation="portrait"
-            android:windowSoftInputMode="adjustResize"
+            android:windowSoftInputMode="adjustNothing"
```

Comment:
This prevents the biometric login (or SP login) button from moving above they keyboard and blocking the UI if the user chooses to type into the webview.